### PR TITLE
Updated packages, fixed build issues

### PR DESCRIPTION
**Pull request by JulianBerger**
- Updated all packages
- Fixed prettier conflicts
- Bumped version to 5.5.1

### CODE REVIEW

**Comment by JulianBerger** [comment]
Ava does not find the tests on AppVeyor ('**/*.test.js does not exist.')...

**Comment by JulianBerger** [comment]
Does someone know why ava cant resolve the test path on appveyor?

**Comment by rhummelmose** [comment]
Why isn't this merged?

**Comment by chris-ritsen** [comment]
Please merge this. I can't build v0.22.1 on node v13.1.0.

**Comment by evilebottnawi** [comment]
Looks @brunocodutra is unavailable, we need new maintainer

**Comment by evilebottnawi** [comment]
Ping me if somebody have time to maintenance

**Comment by chris-ritsen** [comment]
Workaround for my issue: add `"resolutions": {  "sharp": "^0.23.2" }` to `package.json`

**Comment by hisapy** [comment]
Hi devs, is there any estimated time to merge this? in my case I've just started a new repo and I'm noticing errors like the following:

```
 favicons > jimp > @jimp/custom > core-js@2.6.10: core-js@<3.0 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
```

Thanks in advance :)

File: package.json [hunk 1]
```diff
@@ -33,41 +42,44 @@
     "url": "https://github.com/itgalaxy/favicons/issues"
   },
   "homepage": "https://github.com/itgalaxy/favicons",
+  "engines": {
+    "node": ">=8.9.4 <9 || >=10.0.0 <11 || >=12.0.0"
```
Comment (evilebottnawi) [inline package.json]: Simplify this using `>=8.9.4`

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "favicons",
-  "version": "5.5.0",
+  "version": "5.5.1",
```
Comment (evilebottnawi) [inline package.json]: Avoid this in PR, we will do this when start to release

File: appveyor.yml [hunk 3]
```diff
@@ -2,9 +2,10 @@
 
 environment:
   matrix:
-    - nodejs_version: 10
-    - nodejs_version: 8
-    - nodejs_version: 6
+    - nodejs_version: "12"
```
Comment (evilebottnawi) [inline appveyor.yml]: Let's add `13` node

**Comment by gacallea** [comment]
hi :)

here because I get these errors when I try to npm build using responsive-loader and favicons-webpack-plugin together for a prod build. in a npm run dev I have no issues, since I don't do favicons for dev. can sharp be an external dep and use sharp from npm?

```
Eirian:gentlewashrec aa$ npm run build

> gentlewashrec@1.0.0 build /Users/aa/Project and Ideas/IT/Sites/gitlab/onPages/gentlewashrec
> webpack --config webpack.prod.js; docker-compose -f "docker-compose.yml" up -d --build

10% building 0/0 modules 0 active
(sharp:90198): GLib-GObject-WARNING **: 10:44:27.449: cannot register existing type 'VipsObject'

(sharp:90198): GLib-CRITICAL **: 10:44:27.449: g_once_init_leave: assertion 'result != 0' failed

(sharp:90198): GLib-GObject-CRITICAL **: 10:44:27.449: g_type_register_static: assertion 'parent_type > 0' failed

(sharp:90198): GLib-CRITICAL **: 10:44:27.449: g_once_init_leave: assertion 'result != 0' failed
^C
```

please and thank you

**Comment by JulianBerger** [comment]
Now the canvas build does fail for node 13.0.1 on Windows

**Comment by evilebottnawi** [comment]
@JulianBerger maybe we should open issue in `canvas` repo?

**Comment by JulianBerger** [comment]
There is already an issue: https://github.com/Automattic/node-canvas/pull/1485
They just have no prebuild windows binary for node 13.

@evilebottnawi  Should we just remove node 13 from Appveyor for now?!

**Comment by evilebottnawi** [comment]
@JulianBerger yes :+1:

**Comment by evilebottnawi** [comment]
@JulianBerger I think we should do release, ping me tomorrow and i will do it

**Comment by jantimon** [comment]
This pr broke favicons-webpack-plugin #293

**Comment by gacallea** [comment]
@evilebottnawi @JulianBerger 

the issue is still present on Node LTS, was this fix only for node 13?

**Comment by evilebottnawi** [comment]
@gacallea what issue?

**Comment by gacallea** [comment]
this one https://github.com/itgalaxy/favicons/pull/290#issuecomment-559013022 

basically the inclusion of sharp with favicons conflicts with sharp installed separately and breaks favicons generation.

**Comment by evilebottnawi** [comment]
@gacallea Can you open a new issue with reproducible test repo?

**Comment by gacallea** [comment]
@evilebottnawi sure thing. I'll do after lunch.

**Comment by gacallea** [comment]
@evilebottnawi sorry for taking so long, life's a bitch. I've made my repo public for you to check https://gitlab.com/gacallea/gentlewashrec/

**Comment by evilebottnawi** [comment]
@gacallea 

> sorry for taking so long, life's a bitch

yes

In my todo :+1:

**Comment by gacallea** [comment]
@evilebottnawi just one more piece of useful info: 

if I comment out ```// adapter: require('responsive-loader/sharp'),``` on favicons creation, everything runs fine. I think this confirms that the included sharp is the issue here.

**Comment by evilebottnawi** [comment]
Looks like bug in `responsive-loader`

**Comment by gacallea** [comment]
> Looks like bug in `responsive-loader`

how can I test favicons without the included sharp ? wouldn't it be best to not have it there and use the external package?

**Comment by evilebottnawi** [comment]
@gacallea sharp used for generation images, you can't avoid this package

**Comment by gacallea** [comment]
@evilebottnawi I know, but I was wondering if it would be better to use sharp installed with npm rather than including it with favicons deps

**Comment by evilebottnawi** [comment]
@gacallea where we include sharp? it is just the dep

**Comment by gacallea** [comment]
I see. I think I got this wrong then. It must be included in the webpack plugin then. Sorry for confusion, I'm new to this.

```
Eirian:gentlewashrec aa$ npm ls sharp
gentlewashrec@1.0.0 /Users/aa/Project and Ideas/IT/Sites/gitlab/onPages/gentlewashrec
├─┬ favicons-webpack-plugin@1.0.2
│ └─┬ favicons@5.4.1
│   └── sharp@0.22.1 
└── sharp@0.23.4 
```

**Pull request merged**